bk-api/bk_api.py

```diff
@@ -585,17 +585,25 @@ def scp(source, node_id, target):
 
 
 
-def node_ssh(node_id, command, input_str=None):
+def node_ssh(node_id, command, input_str=None,quiet_mode=False):
+    proxy_cmd = f"ProxyCommand=ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@{BEEKEEPER_SSHD_HOST} -p 2201 -i /config/admin-key/admin.pem"
```

Comment:
why dont you put the `-o` in the `proxy_cmd`

```
proxy_cmd = ["-o" , f"ProxyCommand=ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@{BEEKEEPER_SSHD_HOST} -p 2201 -i /config/admin-key/admin.pem"]
```

Reply:
It's easier/simpler to just append to a string when I add `-q` to the proxy_cmd.